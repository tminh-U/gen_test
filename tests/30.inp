10
8 10 1 12 8 5 11 4 5 3 
7
9 9
2 8
1 1
2 8
2 10
1 8
8 8
