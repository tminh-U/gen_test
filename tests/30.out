2
19
2
19
27
27
2
